apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -38,10 +38,10 @@ export class HeftToolsCommandLineParser extends CommandLineParserBase {
     };
 
     const cleanAction: CleanAction = new CleanAction(actionOptions);
-    const buildAction: BuildAction = new BuildAction(actionOptions);
+    const buildAction: BuildAction = new BuildAction({ ...actionOptions, cleanAction });
```

Comment:
Interpolating from a class seems incorrect. What if the class has private members?

Reply:
What do you mean here?

Reply:
Ohh sorry I misread that as `...cleanAction`.